packages/mangohud/mangohud.pacscript

```diff
@@ -1,11 +1,13 @@
 name="mangohud"
 repology=("project: mangohud")
-pkgver="0.7.0"
-url="https://github.com/flightlessmango/MangoHud/releases/download/v${pkgver}/MangoHud-v${pkgver}-Source.tar.xz"
+pkgver="0.7.1"
+
 homepage='https://github.com/flightlessmango/MangoHud'
+url="${homepage}/archive/refs/tags/v${pkgver}.tar.gz"
+
```

Comment:
I don't like this. Add the full URL for this, it kinda obscures it.

Reply:
Done